cmd/root.go

```diff
@@ -121,8 +121,15 @@ func serverConfigurationFromConfig() *core.ServerConfiguration {
 		}
 	}
 
+	var whitelistedExtensions = []string{}
```

Comment:
We should use `allowedExtensions` instead

Reply:
updated